### PR TITLE
WIP: Fix scrapers

### PR DESCRIPTION
**Pull request by tlyon3**
Fixes the following scrapers that broke:

- [x] TXVaccineCountyRace
   - Couple problems. `state_fips` was returning a string. `County Name` is now just `County`. And there are some rows with "Other" as the county name
- [x] TXVaccineCountyAge
    - Same as `TXVaccineCountyRace`
- [x] NCCountyVaccine
    - This scraper doesn't exist. It's under `NCVaccine`. Removed the DAG
- [x] NCVaccine
   - Only problem was trailing whitespace at the end of the county names. Trimmed off whitespace and works
- [x] PennsylvaniaVaccineDemographics
     - This class isn't exported by `scrapers/official/__init__.py`. All the other demographic specific scrapers are running fine. Should this DAG be deleted?
     - Deleted the DAG
- [x] MinnesotaCountyVaccines
   - PowerBi dashboard. @cc7768 should be solving a problem related to PowerBi dashboards
   - Fixed by updating the dashboard url
   - The site has a captcha and prevents the automatic retrieval of the current key. Need to update the url everytime the key changes (about once a week)
- [x] TexasCountyVaccine
    - Some rows had non-country `location_name`'s. Fixed by removing rows that were not a valid county
- [x] CaliforniaVaccineCounty
    - looks like the URL used doesn't point exactly to the dashboard
    - fixed by editing TableuBase class and allowing a URL override
- [ ] WYCountyVaccinations [disabled on frontend]
    - The initial request for the data is returning an error. The error's `reasonStr` is `PREFETCH_VALIDATION`
- [ ] WYStateVaccinations
    - Same as `WYCountyVaccinations`
- [x] GeorgiaCountyVaccine
    - The correct resource on the ArcGIS server seems to have moved. I found the [matching one](https://services6.arcgis.com/t7DA9BjRElflTVpw/ArcGIS/rest/services/Georgia_DPH_COVID19_Vaccination_public_v2_VIEW/FeatureServer/7). There are two different datasets, one with an `ADM` name and another with `RES`. `ADM` isn't returning data, but `RES` is. I'm not sure what the original one was. Maybe @cc7768 can clarify
- [x] VirginiaVaccine (https://trello.com/c/QqObNrn8/1054-vaccine-data-dropped-in-va-because-data-stale)
    - Remote column names changed. Updated code to use new column names
- [x] ArizonaVaccineCounty
  - County columns had `Tribes` as one of the counties. Fixed by removing all rows with `location_name == 'Tribes`
- [x] CovidTrackingProjectDemographics
    - Some of the entries in the `Cases_Black` column were `#REF!`. `normalize` couldn't parse these. Fixed by coercing `to_numeric` on that column. `#REF!` entries are now `NaN`
- [x] MissouriVaccineCounty
   - Column names had changed. Updated column names to match remote data
- [x] HHSReportedPatientImpactHospitalCapacityFacility
    - Bedford City, VA was changed to a town and included in the county fips in 2013. For some reason reports still have the old City FIPS. Fixed by replacing the old City FIPS with the corresponding county FIPS it belongs to (edited)

### CODE REVIEW

**Comment by igorkofman** [comment]
Looks great - let's ship it!

**Comment by sglyon** [comment]
Thanks @tlyon3 !!

I just pulled and ran tests locally.

I'm seeing that `HHSReportedPatientImpactHospitalCapacityState` is also failing. I know this wasn't on the original list, but will you please take a look.

Also it looks like `ArizonaVaccineCounty` is failing for me locally. Will you take a look there too?

**Comment by sglyon** [comment]
Going to mere this now, we will work on WY in another place

**Review by sglyon — APPROVED** [review]

File: can_tools/scrapers/official/AZ/az_vaccine.py [hunk 1]
```diff
@@ -76,6 +79,11 @@ def normalize(self, data) -> pd.DataFrame:
                 measurement="cumulative",
                 unit="doses",
             ),
+            "janssen_vaccine_allocated": CMU(
+                category="janssen_vaccine_allocated",
```
[the remark this answers is not in the document]
Comment (tlyon3) [reply can_tools/scrapers/official/AZ/az_vaccine.py:83]: The other vaccines that were in the original code were both allocated. I don't think the data returned from the fetch has given, only allocated

**Pull request merged**